clients/admin/src/error.rs

```diff
@@ -0,0 +1,12 @@
+use backtrace::Backtrace;
+use std::fmt::Debug;
+
+pub struct Error;
```

Comment:
I think playing around with backtraces inside errors is going to be useful for the future